design/AstroMancer Cards.md

```diff
@@ -0,0 +1,21 @@
+##AstroMancer Cards
+
+###AstroMancer Cards can be played for 2 Mana
+
+Starfall -2 Mana Quadra-cast(Total cost = 2) Deal 6 damage to a random enemy. Feed: Heal a random ally 2.(So 8 healing in all.)
+
+Star Slash +1 Mana deal .5 damage to an enemy. Feed: Heal 1 for each mana.
+
+Starlight -1 Mana Deal 5 damage to all enemies. Feed: The Solar card has +10% spell Power for each enemy hit.
+
+Sunburn -2 Mana Deal 12 damage to all unshielded characters. Feed: 20% Chance to recast.
```

Comment:
Do we want to have any flat damage effects? Flat damage seems like it would impair the ability to make stats meaningful...

Reply:
So I want to make a distinction between magical and physical. The distinction is this. Physical things are affected by the move you are performing. So a card will be making you do 120% Attack damage. So you have a flat amount of strength(say 11) which is applied to your weapon(say your weapon makes it hit at .9 but also has .1 lifesteal) then the action modifies it by a %.(So in this case you would attack for 12[11.88]) and heal 1.) Notice how the flat number is your strength.

For Magical effects, I want the strength to be inherent to the spell, and that is modified by the power of the caster. So let's resolve Sunburn. I have already played Starlight hitting 3 enemies. So Sunburn would do 12, but since I have 30% Spell Power from Starlight & 15% Spell Power from my stats, I am going to deal 17-18 damage to all unshielded characters. So this formula just changes which number is flat.